### ISSUE TITLE
Проверка раздачи в transmission

### ISSUE THREAD

**Issue by mdevaev** [mdevaev/emonoda#22]
Сообщение от юзера:

> Походу rtfetch не смотрит есть ли раздача в трансмиссии, а берет из своих данных, что раздача когда-то была, и если она обновилась, то он ее перекачивает заново. По другому я не могу объяснить как у меня появляется раздача, которую я удалил.

Проверить.

**Comment by mdevaev** [comment]
@dganic давайте еще раз, по пунктам, тезисно и подробно. Когда проявляется, почему?

**Comment by dganic** [comment]
Этот баг у меня проявляется при перезагрузки transmission, большинство обновляемых раздач добавляются в загрузку и начинают качаться в дефолтный каталог загрузки, а не по тем путям где расположены файлы. Возможно это как-то связано с каталогом resume:   https://trac.transmissionbt.com/wiki/ResumeFile

**Comment by mdevaev** [comment]
Так, а в вашем resume-каталоге действительно лежит информация об уже удаленных раздачах?

**Comment by dganic** [comment]
Ну не удаленных, а обновленных через rtfetch,. Да лежат.

**Comment by mdevaev** [comment]
Старых версий обновленных раздач, так что ли?

**Comment by dganic** [comment]
Да все верно.

**Comment by mdevaev** [comment]
Окей, я сделал следующее:
* Добавил торрент сериала месячной давности;
* Обновил его через emfetch;
* Во время скачки перезапустил transmission-daemon
* Увидел только обновленную раздачу.
Что я делаю не так?

**Comment by dganic** [comment]
Скриншот transmission до перезагрузки: http://storage1.static.itmages.ru/i/15/0413/h_1428949768_9236964_8ee0c1f06a.png

Листинг директории resume: https://paste.kde.org/peakqntfz

Скриншот после перезагрузки transsmission: http://storage3.static.itmages.ru/i/15/0413/h_1428949806_6941545_3f6486d3b2.png
Видно что качается в дефолтную директорию /media/pomoika
Обычно больше, на днях просто изрядно почистил торрент лист.

**Comment by mdevaev** [comment]
Какая версия transmission,

**Comment by mdevaev** [comment]
Говорят, что это баг в трансмишне и в 2.84 его уже точно нет.

**Comment by dganic** [comment]
Из extra archlinux 2.84

**Comment by fangelion** [comment]
Debian 8.0 версия трансмиссии  2.84 только что ковырялся, бага нет.
ЗЫ. про баг я звенел.

**Comment by mdevaev** [comment]
Может, дело в каких-нибудь хитрых настройках?

**Comment by mdevaev** [comment]
В общем, не знаю. У меня не воспроизводится, у @fangelion тоже не воспроизводится, но проблема имеет место быть. Может, сами попробуете отладить?

**Comment by dganic** [comment]
Всё утро сегодня пытался отловить баг :) Первое что сделал перевел всю работу юнита emonoda  на работу от пользователя transmission, т.к как согласно вики: https://wiki.archlinux.org/index.php/Transmission у меня трасмишион работает именно от этого пользователя.  Раньше  в директории хранения файлов торрент файлы обновленные emonoda сохранялись от   имени root: http://storage2.static.itmages.ru/i/15/0418/h_1429346540_2506919_f8ee8d2fdb.png Мне казалось, что этот баг возникает из-за проблем с правами на торрент файлы.  Но после перевода работы юнита на работу от имени transmission торрент файлы стали сохраняется с нужным владельцем, но при сохранение нового файла в директорию трансмишена, старый фаил не удаляется а перезаписывается самим сабой ( возможно из директори с бекап торрент файлами?) Видно это на этом скрине: http://storage1.static.itmages.ru/i/15/0418/h_1429347067_5207469_8a0bf6c0ef.png на раздаче "Реальные пацаны"  время создания у нового и старого файла одинаковое: 11:16
Конфиг transmission: https://paste.kde.org/pczt18x0f

**Comment by mdevaev** [comment]
Честно говоря, у меня нет идей, как это лечить. Я не смог воспроизвести. Будем ждать кого-нибудь со знаниями питона и таким же багом, чтобы он сам мог поправить.

**Comment by dganic** [comment]
Я бы дал доступ к серваку, но я за натом, может образ системы сделать, или может в emonoda какой-либо расширеный дебаг включить?

**Comment by mdevaev** [comment]
Более подробных дебагов нет, пишется все, что нужно.

**Comment by ASMfreaK** [comment]
У меня проявляется эта проблема. Регулярно после emupdate появляются
```
=== Torrent not in client (1):
```
И увеличиваются в количестве с каждым обновлением

**Comment by mdevaev** [comment]
@ASMfreaK этот баг - уже притча во языцех :( На моей инсталляции у меня вообще никак не воспроизвелось, и никакая диагностика не помогала. Вы же умеете питон, может, попробуете как-то отладить?
